### DISCUSSION TITLE
Melhor estratégia para uso do appsettings.json

### DISCUSSION THREAD

**Discussion by rafaelevoinfo** [balta-io/forum#261] [Tira Dúvidas] [answered]
No curso de Fundamentos do ASP.NET 6 na aula Configurações de Debug é falado sobre os arquivos appsettings.json e appsettings.Development.json, porém qual seria a melhor estratégia quando estamos trabalhando em um projeto de código fonte aberto? Pois podemos ter informações sensíveis nesses arquivos e não podemos comita-los. Por outro lado, se não adicionarmos as informações a esses arquivos outros programadores teriam dificuldade em saber quais configurações eles precisam preencher.
Outra dúvida seria na hora de fazer o deploy, deveríamos colocar o arquivo appsettings.json no servidor? Pergunto porque já li algumas recomendações onde dizem para não colocar informações sensíveis em arquivos e sim em variáveis de ambiente.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @rafaelevoinfo, como vai?

Nunca devemos enviar informações sensíveis para o GitHub (Ou qualquer outro versionador). Se for a ConnectionString do LocalHost, tudo bem, agora fora o ambiente dev, nada!

A melhor estratégia é seguir o que passo no curso de DevOps com Azure e GitHub, onde mostro como deixar o AppSettings direto no Azure 🚀
